src/box/decimal.h

```diff
@@ -515,19 +515,22 @@ box_decimal_mp_decode(box_decimal_t *dec, const char **data);
  * Decode a decimal from msgpack @a data without the msgpack
  * extension header.
  *
- *  | box_decimal_mp_decode_data() must be called for this position
- *  |                                        |
- *  |                                        v
- *  | <msgpack type> <size> <extension type> <data>
- *  | ^
- *  | |
- *  | box_decimal_mp_decode() must be called for this position
+ * \code
+ *  box_decimal_mp_decode_data() must be called for this position
+ *                                         |
+ *                                         v
+ *  <msgpack type> <size> <extension type> <data>
+ *  ^
+ *  |
+ *  box_decimal_mp_decode() must be called for this position
+ * \endcode
```

Comment:
Nit: It seems the one whitespace indent doesn't affect anything, so I would eliminate it. Or it is by intention?

Reply:
it was an attempt to visually highlight code block in a comment, removed indentation to make consistent with other comments with code blocks